.github/workflows/build-commit.yaml

```diff
@@ -1,6 +1,24 @@
 name: build-commit
 
 on:
+  workflow_call:
+    inputs:
+      arch:
+        description: The machine architecture to build for
+        required: true
+        type: string
```

Comment:
Probably `choice` instead like your other workflow?

Reply:
Actually, `choice`s aren't allowed in `workflow_call`. They're only allowed in `workflow_dispatch`. The types allowed in `workflow_call` are only `bool, string, number`.

Reply:
Relevant docs:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype